main.py

```diff
@@ -344,9 +425,12 @@ def __init__(self, *args):
 		self.inst = Xqemu()
 		self.settings = SettingsManager()
 		self.settings.load()
-		self.runButton.setText('Start')
-		self.pauseButton.setText('Pause')
-		self.screenshotButton.setText('Screenshot')
+		self.libraryCache = CacheManager()
+		self.libraryCache.load()
+		self.gamesCache = []
+
+		# Disable resizing because it doesnt really do anything
+		self.setFixedSize(540, 293)
```

Comment:
Disabling resizing isn't an option in my opinion.
This looks like a cruel hack, rather than good UI design.